README.md

```diff
@@ -22,24 +22,40 @@ It is based on Umee's Gravity Bridge implementation, [Peggo](https://github.com/
 
 ### Dependencies
 
-Building requires [Go 1.17+](https://golang.org/dl/).
-
 Initialize git submodules, needed for Forge dependencies:
 
 ```sh
 git submodule init
 git submodule update
 ```
 
-To regenerate the Go ABI wrappers with `make gen`, you need the `abigen` tool. Install it with:
+To regenerate the Go ABI wrappers with `make gen`, you need the `abigen` tool.
+Building requires [Go 1.17+](https://golang.org/dl/).
+Install `abigen` with:
 
 ```sh
 git clone https://github.com/ethereum/go-ethereum.git
 cd go-ethereum
 make devtools
 ```
 
-### Building
+### Build and Test Contracts
+
+Build with:
+
```

Comment:
```suggestion
```

Reply:
https://github.com/DavidAnson/markdownlint/blob/2d19c0685bfa98dce87f493a8fcffbb5939af2c3/doc/Rules.md#md031---fenced-code-blocks-should-be-surrounded-by-blank-lines